### ISSUE TITLE
Java 11 ?

### ISSUE THREAD

**Issue by rider87** [stephanrauh/BootsFacesDesktopApplication#2]
Hi, 
What changes needs to be done to run via Java11.
If I try to upgrade to JAVA11 I got this error:

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/9787533/178758298-40c4124f-68e6-449a-95e3-12111676bf2a.png">

Any ideas?

**Comment by rider87** [comment]
SCHWERWIEGEND: Allocate exception for servlet Faces Servlet
java.lang.IllegalStateException: Keine Factory als Backup für javax.faces.context.FacesContextFactory gefunden.

**Comment by stephanrauh** [comment]
Not really. But I hope this StackOverflow article sheds some light. "Backup" means "implementation". I didn't expect that, but maybe it helps.

https://stackoverflow.com/questions/25369903/java-lang-illegalstateexception-could-not-find-backup-for-factory-javax-faces

**Comment by stephanrauh** [comment]
BTW, when you've figured it out, feel free to send me a pull request. I've reluctantly set BootsFaces to "maintenance mode" a couple of years ago, but it's still my pet project, so I'm ready to test and merge your PR.
